skcosmo/decomposition/_kernel_pcovr.py

```diff
@@ -160,47 +174,46 @@ class KernelPCovR(_BasePCA, LinearModel):
     >>> Y = np.array([[ 0, -5], [-1, 1], [1, -5], [-3, 2]])
     >>> Y = SFS(column_wise=True).fit_transform(Y)
     >>>
-    >>> kpcovr = KernelPCovR(mixing=0.1, n_components=2, kernel='rbf', gamma=2)
+    >>> kpcovr = KernelPCovR(mixing=0.1, n_components=2, regressor=KernelRidge(kernel='rbf', gamma=2), kernel='rbf', gamma=2)
     >>> kpcovr.fit(X, Y)
-        KernelPCovR(coef0=1, degree=3, fit_inverse_transform=False, gamma=0.01, kernel='rbf',
-           kernel_params=None, mixing=None, n_components=2, n_jobs=None,
-           alpha=None, tol=1e-12)
+        KernelPCovR(gamma=2, kernel='rbf', mixing=0.1, n_components=2,
+                    regressor=KernelRidge(gamma=2, kernel='rbf'))
     >>> T = kpcovr.transform(X)
-        [[ 1.01199065, -0.35439061],
-         [-0.68099591,  0.48912275],
-         [ 1.4677616 ,  0.13757037],
-         [-1.79874193, -0.27232032]]
+        [[-0.55119827, -0.21793572],
+         [ 0.3768726 ,  0.31208068],
+         [-0.76898956,  0.08511876],
+         [ 0.92488574, -0.18627707]]
```

Comment:
Why do the result change so drastically if the data stays the same?

Reply:
The example was broken as written. You'll notice that the (old) example input initializes with `gamma=2`, but the corresponding example output shows `gamma=0.01`. So I ran the example myself with `gamma=2`, and this is what came out. I can try with `gamma=0.01` to see if the results are the same as the old.